.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
```

Comment:
v2 is current, not v1. Should probably also just use current JDK  (unless we want to set up a matrix for multi-version testing later), it's perfectly capable of targeting older versions.

```yaml
      - name: Set up JDK
        uses: actions/setup-java@v2
        with:
          distribution: 'adopt'
          java-version: '17'
```